hyrax/db/schema.rb

```diff
@@ -12,7 +12,10 @@
 
 ActiveRecord::Schema.define(version: 2020_01_08_111832) do
 
-  create_table "bookmarks", force: :cascade do |t|
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
```

Comment:
this and the `id: :serial` are because the schema was generated from a postgersql install instead of a sqilte3 install.  Sqlite3 doesn't have automatic id serialization built in, Rails does it for you, but that's a difference between local and production